apple/internal/transition_support.bzl

```diff
@@ -172,7 +170,7 @@ def _cpu_string(*, environment_arch, platform_type, settings = {}):
         visionos_cpus = settings["//command_line_option:visionos_cpus"]
         if visionos_cpus:
             return "visionos_{}".format(visionos_cpus[0])
-        return "visionos_x86_64"
+        return "visionos_sim_arm64"
```

Comment:
i think this if condition is redundant? i don't think it will ever make it here since the condition above is the same? must have been my fault but can you remove it?

Reply:
Done